src/plugins/ejs-engine.ts

```diff
@@ -0,0 +1,35 @@
+import type {MojoContext} from '../types.js';
+import type {AsyncTemplateFunction} from 'ejs';
+import {compile} from 'ejs';
+import type App from '../app.js';
+import {createHash} from 'crypto';
+import File from '../file.js';
+
+export default function ejsEnginePlugin (app: App): void {
+  app.renderer.addEngine('ejs', ejsEngine);
+}
+
+async function ejsEngine (ctx: MojoContext, options: {inline: string, viewPath: string}): Promise<Buffer> {
```

Comment:
The options are wrong, we are going to need a `MojoEngineOptions` in `types.js` with all the possible ones as optional.